src/game_character.h

```diff
@@ -23,6 +23,10 @@
 #include "color.h"
 #include "rpg_moveroute.h"
 
+#ifdef __MORPHOS__
+#undef Wait
```

Comment:
Where is all that namespace pollution coming from?
Is there a way to solve this in EasyRPG or is the system library the problem?

Reply:
It's coming from a system header for exec.library. Shared library functions are defined as macros, and they often end up included by C library headers, causing namespace clashes like this.